eth-contracts/contracts/DelegateManager.sol

```diff
@@ -799,6 +819,18 @@ contract DelegateManager is InitializableV2 {
         return spDelegateInfo[_sp].delegators;
     }
 
+    /**
+     * @notice Get total delegation
+     * @param _delegator - delegator address
+     */
+    function getTotalDelegation(address _delegator)
```

Comment:
`getTotalDelegatedStakeForDelegator()` ? "getTotalDelegation" is unclear

Reply:
going with `getTotalDelegatedStake(address delegator)`